bombastic/walker/src/lib.rs

```diff
@@ -84,6 +84,10 @@ pub struct Run {
     /// Additional root certificates for the destination
     #[arg(long = "sender-root-certificates")]
     pub additional_root_certificates: Vec<PathBuf>,
+
+    /// Allow logging of uploaded sbom file reports.
+    #[arg(long, env, default_value_t = true, action = ArgAction::Set)]
+    pub is_report: bool,
```

Comment:
(non-blocking): I am not a big fan of the name of this (same for the other locations). Maybe `report-enabled`? Or just `--report` (and using `.report.enabled` as the chart variable?

Reply:
ok I will change to "report-enabled"